### PR TITLE
Refactoring Styles

### PR DESCRIPTION
**Pull request by Mi-Lan**
- We need to implement this colors and restructure the current theme index file

### CODE REVIEW

File: app/src/theme/index.tsx [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+import { ThemeProvider as StyledThemeProvider } from 'styled-components'
+
+import { Colors } from './types'
+
+export function colors(darkMode: boolean): Colors {
+  return {
+    // text
+    text1: darkMode ? '' : '#37474F',
+    text2: darkMode ? '#C3C5CB' : '#86909E',
+    text3: darkMode ? '#6C7284' : '#86909E',
```
Comment (Frankaus) [inline app/src/theme/index.tsx:11]: @Mi-Lan  text2 and text3 have the same value on the code, mistake? figma has a different value for text3
Screenshot 2021-06-22 at 16 16 08

**Review by Frankaus — APPROVED** [review]
I could create markets, deposit, withdraw, buy, sell and request Kleros verification. Works fine here:)

**Review by pimato — CHANGES_REQUESTED** [review]
After creating a market and letting it finalized it seems the app is crashing trying to open this link:

http://localhost:3000/#/0x638dd4a95d8cc3e70aaa2fb3d700ab936a391c8e/buy

error message:
Bildschirmfoto 2021-08-02 um 18 27 18

**Comment by Mi-Lan** [comment]
@pimato Hey men found the issue! It was some small issue related to the market_buy on refactoring branch...Should work now!

**Comment by pimato** [comment]
- [ ] Categorical markets
    - [ ] Create market
    - [ ] Buy
    - [ ] Sell
    - [ ] Deposit
    - [ ] Withdraw
    - [ ] Bond outcome
    - [ ] Redeem winnings
- [ ] Scalar markets
    - [ ] Create market
    - [ ] Buy
    - [ ] Sell
    - [ ] Deposit
    - [ ] Withdraw
    - [ ] Bond outcome
    - [ ] Redeem winnings
    - [ ] Position table works as expected
        - Correct payout/profit/loss
    - [ ] Market buy slider works as expected
        - Correct payout/profit/loss
- [ ] Account
    - [ ] Correct wallet balances
    - [ ] Correct omen account balances
- [ ] xDai relay
    - [ ] Deposit
    - [ ] Withdraw
- [ ] Mobile
    - Should just be relatively functional as per usual

**Comment by pimato** [comment]
looks like the `Sell`/ `Buy` Button on a finalized markets has not the disabled text color anymore:
Bildschirmfoto 2021-08-03 um 16 41 38

should:
Bildschirmfoto 2021-08-03 um 16 42 19

**Review by pimato — APPROVED** [review]

**Comment by Frankaus** [comment]
- [ ]  Categorical markets
  - [ ]  Create market
  - [ ]  Buy
  - [ ]  Sell
  - [ ]  Deposit
  - [ ]  Withdraw
  - [ ]  Bond outcome
  - [ ]  Redeem winnings
- [ ]  Scalar markets
  - [ ]  Create market
  - [ ]  Buy
  - [ ]  Sell
  - [ ]  Deposit
  - [ ]  Withdraw
  - [ ]  Bond outcome
  - [ ]  Redeem winnings
  - [ ]  Position table works as expected
  - [ ]  Market buy slider works as expected

**Pull request merged**